portfolio/src/main/java/com/google/sps/classes/Comment.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.classes;
+
+public class Comment {
+    private String username;
+    private String comment;
+    private long timestamp;
+
+    public Comment(String _username, String _comment, long _timestamp) {
+        username = _username;
+        comment = _comment;
+        timestamp = _timestamp;
+    }
```

Comment:
nit: prefixing with `_` is not a very common practice in Java. The more idiomatic way to write this constructor in Java would be
```java
    public Comment(String username, String comment, long timestamp) {
        this.username = username;
        this.comment = comment;
        this.timestamp = timestamp;
    }
```

as in [this example](https://github.com/oscardavidrm/step/blob/master/walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/data/ServerStats.java)